pycloudlib/kvm/cloud.py

```diff
@@ -49,48 +50,88 @@ def get_instance(self, instance_name):
         """
         return KVMInstance(instance_name)
 
-    def launch(self, name, release, inst_type=None, wait=True):
```

Comment:
Given that you are diverging from distribution/release aliases for a given cloud to a specific launch by image_id approach, how do you suggest we handle these cloud-specific lookups of proper image_id aliases for a given Ubuntu release?

Reply:
It would be in the implementation of `released_image` or `daily_image`. I.e.,

```python
instance = client.launch(image_id=client.released_image('focal'))```

Reply:
ahh right. Thanks, brain fail.+1